src/index.html

```diff
@@ -1,16 +1,59 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
remove blank lines between ALL parent and child tags
```suggestion
<html lang="en">
  <head>
```